plugins/Makefile.mk

```diff
@@ -23,3 +23,8 @@ include plugins/dmx4linux/Makefile.mk
 include plugins/e131/Makefile.mk
 include plugins/uartdmx/Makefile.mk
 endif
+
+dist_noinst_SCRIPTS += plugins/convert_README_to_header.sh
+
+plugins/%/PluginDescription.h: plugins/%/README.md plugins/convert_README_to_header.sh plugins/Makefile.mk
```

Comment:
Travis is currently logging the following:
```
plugins/Makefile.mk:29: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:155:   'plugins/Makefile.mk' included from here
plugins/Makefile.mk:30: warning: shell dirname $<: non-POSIX variable name
plugins/Makefile.mk:30: (probably a GNU make extension)
```

So I'd be tempted to switch to a more long-hand method and put the config in each plugin's Makefile.mk.